src/matrixHelpers/formioLoader/index.js

```diff
@@ -0,0 +1,166 @@
+const initFormioInstance = (formioElem, opts) => {
```

Comment:
do you have a storybook story for this so i can check it out outside of matrix

Reply:
good idea, will create story example for this